code/muilty_rtspsender_gui/rtsp_server.py

```
@@ -0,0 +1,1830 @@
+            batch_content += "    echo    MediaMTX를 다운로드하고 PATH에 추가하거나\n"
+            batch_content += "    echo    이 배치 파일과 같은 폴더에 넣어주세요.\n"
+            batch_content += "    echo    다운로드: https://github.com/bluenviron/mediamtx/releases\n"
+            batch_content += "    pause\n"
+            batch_content += "    exit /b 1\n"
+            batch_content += ")\n"
+            batch_content += "echo ✅ MediaMTX 실행 파일을 찾았습니다.\n"
+            batch_content += "echo.\n"
+            
+            # 설정 파일 확인
+            batch_content += "echo [INFO] 설정 파일들 확인 중...\n"
+            for i, config_file in enumerate(config_files):
```

Ruff (0.12.2)

Loop control variable `i` not used within loop body

(B007)